### ISSUE TITLE
`Logger.broadcast` removed from ActiveSupport causing sidekiq to break

### ISSUE THREAD

**Issue by kjellberg** [sidekiq/sidekiq#6054]
Ruby version: 3.2
Rails version: master

#### The problem
When attempting to run Sidekiq with the latest Rails master branch, the following error is encountered:

```console
➜  core git:(main) ✗ bundle exec sidekiq
undefined method `broadcast' for ActiveSupport::Logger:Class
/Users/rasmus/.asdf/installs/ruby/3.2.0/lib/ruby/gems/3.2.0/gems/sidekiq-7.1.4/lib/sidekiq/rails.rb:52:in `block (2 levels) in <class:Rails>'
/Users/rasmus/.asdf/installs/ruby/3.2.0/lib/ruby/gems/3.2.0/gems/sidekiq-7.1.4/lib/sidekiq.rb:98:in `configure_server'
/Users/rasmus/.asdf/installs/ruby/3.2.0/lib/ruby/gems/3.2.0/gems/sidekiq-7.1.4/lib/sidekiq/rails.rb:47:in `block in <class:Rails>'
```

This issue seems to be related to changes in ActiveSupport, specifically the removal of the broadcast method in ActiveSupport::Logger. 

#### Steps to reproduce:

1. Use the latest Rails master branch.
2. Attempt to run Sidekiq.

#### Related PR:
See https://github.com/rails/rails/pull/48615

**Comment by fatkodima** [comment]
Encountered the same problem in my gem - https://github.com/fatkodima/online_migrations/commit/9460c2b19d12d2bf7ab74fb0b0c7cc818b670e8d.

**Comment by drale2k** [comment]
Having this issue with `7.1.0.rc1`, which means this will make it into the 7.1 release. Is there any quick workaround from Sidekiqs side, so i don't have to downgrade Rails?

**Comment by billyabm** [comment]
in my case, I solved this issue using sidekiq`7.2.2`
```
gem 'sidekiq', '~> 7.2.2'
```
